### PR TITLE
Profile page users/show (events and events you have rsvp'd show in same table); updated homepage test's users/guest

### PR DESCRIPTION
**Pull request by brobs0111**
Deleted other partial and variable.

### CODE REVIEW

**Comment by RogelioMart** [comment]
TL/DR: The new feature works great but there are errors when signing up, the placement for the password error message should be corrected, and duplicate activism halls can be created. Do not merge with master

Report
The new feature works great I can see what stuff I have rsvp'd to in my view profile page. The only issues are that when I tried to sign I first tried signing up with a password that did not have enough characters. The proper error message appeared but it did not have the right placement. 
blackTest_error message not displaying in its totality

Afterwards I tried signing up with the correct information but my local host showed the following error. 

tried_to_sign_up

I got past this by modifying the database to force a user in.
Also I was able to add in duplicate Activism halls with the same hour, date, and name. 

Screenshot from 2020-07-23 16-25-00


I do not recommend merging this code with master until those three errors are fixed.

Personal opinion: The new feature that lets me see what I rsvp to is great but it does not show the names of the events it just shows the topic. This may be troublesome for users because they might not remember what kind of conversation they signed up to. This can be easily fixed by putting the name of the event in the rectangle that shows up in the view profile page.

**Comment by ece106** [comment]
Please put the title of the mystery events on the user's schedule.

**Comment by RogelioMart** [comment]
Retested it. I works great its only problem is that I see the events that I rsvp'd to and none of them have names just dates. This will be confusing to users. Also when rsvping to recurring events it rsvp's me to all of the recurring events not just the one I wanted.

Do not merge with master yet, there are a few issues.

**Comment by ece106** [comment]
The multiple RSVPs is an issue with @mohamedAlmukhtar 's feature. I hope he's working on that.
